.yamato/all-vfx_urp.yml

```diff
@@ -7,19 +7,45 @@
 
 ---
 
-All_VFX_URP_2020.2:
-    name: All VFX_URP CI - 2020.2
+PR_VFX_URP_2020.2:
+    name: VFX_URP PR Job - 2020.2
     dependencies:
-      -  path: .yamato/vfx_urp-win-dx11.yml#VFX_URP_Win_DX11_Standalone_mono_Linear_2020.2
+      -  path: .yamato/vfx_urp-win-dx11.yml#VFX_URP_Win_DX11_Standalone_cache_mono_Linear_2020.2
          rerun: on-new-revision
-      -  path: .yamato/vfx_urp-win-dx11.yml#VFX_URP_Win_DX11_playmode_mono_Linear_2020.2
+    triggers:
+        expression: pull_request.target eq "master" AND NOT pull_request.draft AND pull_request.push.changes.all match ["com.unity.visualeffectgraph/**/*", "TestProjects/VisualEffectGraph_URP/**/*"]
```

Comment:
Typo here too but with `master` (I think all expression triggers should be doubled checked)

Reply:
thanks!

Reply:
now im worried that its like this in 2021.1/staging :O